app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
It's a good practice to use type annotation for all parameters. Here, it would be useful to know what type of objects are expected in the 'friends' list. For example, if it's a list of 'Friend' objects, you could annotate it like this: 'friends: List[Friend]'.